src/sentry/release_health/metrics.py

```diff
@@ -606,3 +601,342 @@ def extract_row_info(row: Mapping[str, Union[OrganizationId, str]]) -> ReleaseNa
             return reverse_tag_value(organization_id, row.get(release_column_name))  # type: ignore
 
         return {extract_row_info(row) for row in result["data"]}
+
+    @staticmethod
+    def _get_session_duration_data_for_overview(
+        where: List[Union[BooleanCondition, Condition]], org_id: int
+    ) -> Mapping[Tuple[int, str], Any]:
+        """
+        Percentiles of session duration
+        """
+        rv_durations: Mapping[Tuple[int, str], Any] = {}
+
+        release_column_name = tag_key(org_id, "release")
+        aggregates: List[SelectableExpression] = [
+            Column(release_column_name),
+            Column("project_id"),
+        ]
+
+        for row in raw_snql_query(
+            Query(
+                dataset=Dataset.Metrics.value,
+                match=Entity("metrics_distributions"),
```

Comment:
Can we add an enum here?

Reply:
we have one, wasn't just merged at the time I wrote this, will fix